src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -283,6 +284,12 @@ export const WorkspaceDashboard = _.flow(
             storageCostEstimate
           ])
         ]),
+        div({ style: styles.header }, ['Owners']),
+        _.map(email => {
+          return div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis' } }, [
+            link({ href: `mailto:${email}` }, [email])
```

Comment:
If possible, it would be great for this link to open in a new tab.

Reply:
`mailto` links are generally handled by an external program, so the experience could vary based on the person's configuration. I think it's better to keep the default behavior in this case.